egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+num_epochs=100
```

Comment:
Ok, thank you. Currently it terminates solely based on number of epochs. But anyway I will keep a small held-out subset for diagnostic (similar to train_tdnn.sh) to see the generalization performance.